scripts/rabbitmq-server.ocf

```diff
@@ -299,7 +299,7 @@ rabbit_stop() {
         return $OCF_SUCCESS
     fi
 
-    $RABBITMQ_CTL stop ${RABBITMQ_PID_FILE}
+    rabbitmqctl_action stop
```

Comment:
I agree, this looks like an oversight. `${RABBITMQ_PID_FILE}` is used with e.g. `rabbitmqctl wait` in other places, for example.